app/upgrades/v0.0.11.go

```diff
@@ -0,0 +1,99 @@
+package upgrades
+
+import (
+	"context"
+
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	cosmosTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/pokt-network/poktroll/app/keepers"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	tokenomicstypes "github.com/pokt-network/poktroll/x/tokenomics/types"
+)
+
+// Upgrade_0_0_11 is the upgrade handler for v0.0.11 Alpha TestNet upgrade
+// Beta TestNet was launched with v0.0.11, so this upgrade is exclusively for Alpha TestNet.
+//   - Before: v0.0.10
+//   - After: v0.0.11
+var Upgrade_0_0_11 = Upgrade{
+	PlanName: "v0.0.11",
+	CreateUpgradeHandler: func(mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+
+		// Adds new parameters using ignite's config.yml as a reference. Assuming we don't need any other parameters.
+		// https://github.com/pokt-network/poktroll/compare/v0.0.10...v0.0.11-rc
```

Comment:
Is this still an rc?

Reply:
Yes, and always will be. We needed the diff between v0.0.10 and v0.0.11-rc. This is a historical code. We don't even need it in the main branch. But I want to preserve to use as a reference.